admin_guide/managing_networking.adoc

```diff
@@ -1121,21 +1136,32 @@ spec:
       port: 5432
 ----
 
-This will allow routers to access the service, but will allow pods in other users'
-namespaces to access it as well. (In general, this should not be a problem though, since
-those pods could normally just access the service via the public router anyway.)
+This allows routers to access the service, but will also allow pods in other
+users' namespaces to access it as well. This should not be a problem though,
+since those pods could normally access the service by using the public router.
+
+Alternatively, you can create a policy allowing full access from the default namespace, as in the *ovs-multitenant* plugin:
```

Comment:
s/plugin/plug-in

Reply:
@gaurav-nelson there's a few more cases of this in your other edits as well. Otherwise, LGTM.

Reply:
@bfallonf IBM Guide says `plugin` is the correct.

Reply:
IBM style guide lists `plugin` for both noun and adjectives. I have changed other instances of `plug-in` to `plugin` as well.